.claude/rules/README.md

```
@@ -0,0 +1,93 @@
+```bash
+# Format and lint
+leptosfmt .
+cargo clippy -- -D warnings
+
+# Pre-commit check
+leptosfmt --check . && cargo clippy -- -D warnings && cargo build
+```
+
+### File Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)